_locales/en/messages.json

```diff
@@ -95,6 +95,16 @@
         }
       }
     },
+    "changedRightAfterVerify": {
+      "message": "Your verification has been cancelled; please review your new safety number with $name$. Remember, this change could mean that someone is trying to intercept your communication or that $name$ has simply reinstalled Signal.",
```

Comment:
Definitely struggled a bit with that one. Thanks for the suggested improvement. :0)